### ISSUE TITLE
Radar.setForegroundServiceOptions missing

### ISSUE THREAD

**Issue by khangaridb** [radarlabs/capacitor-radar#50]
Hello, I saw migration guide from android here from 3.3 to 3.4
https://github.com/radarlabs/radar-sdk-android/blob/develop/MIGRATION.md

Also I saw a commit that bumping android version up to 3.4, I think you have missed integrating Radar.setForegroundServiceOptions to the plugin

**Comment by nickpatrick** [comment]
Hey @khangaridb, thanks for this! We'll investigate and ship a fix tonight.

**Comment by nickpatrick** [comment]
@khangaridb, thanks again for this. The fix is in #51 which we just released with `3.4.0`.

Use like this:

```
Radar.setForegroundServiceOptions({
  options: {
    title: 'Foreground service title',
    text: 'Foreground service text'
  }
});
```

and then

```
Radar.startTrackingContinuous();
```

or

```
Radar.startTrackingCustom({
  options: {
    foregroundServiceEnabled: true,
    ...
  }
});
```

![image](https://user-images.githubusercontent.com/245210/171782609-091f4194-0c4a-43c9-ab3c-3114b28be776.png)

I understand that your team is in touch with our team outside of GitHub, so feel free to reach out directly with any more questions or issues and we'll jump on them ASAP.

Thanks again!

@gianiek @brettguenther
